Dockerfile.template

```diff
@@ -8,12 +8,12 @@ ENV FSXA_NAVIGATION_SERVICE=...
 ENV FSXA_MODE=...
 ENV FSXA_TENANT_ID=...
 COPY package*.json ./
-RUN npm ci
+RUN npm ci --no-optional
```

Comment:
changes in the Dockerfile.template should be backed by the exact same changes in the Dockerfile in our CI. Otherwise the repository template and the CI plans are getting out of sync, which is unwanted.

Reply:
TNG-1167 will solve this